<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstRun</class>
 <widget class="QDialog" name="FirstRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>325</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Witaj!</string>
  </property>
  <widget class="QComboBox" name="First_LanguageSelect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentText">
    <string notr="true">Polski</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string notr="true">Polski</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/flags/Resources/Flags/pl.png</normaloff>:/flags/Resources/Flags/pl.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">English</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/flags/Resources/Flags/en.png</normaloff>:/flags/Resources/Flags/en.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Witaj w programie AutoSwitch!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="First_NextButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>276</y>
     <width>384</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Dalej</string>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/Resources/Icons/accept.png</normaloff>:/icons/Resources/Icons/accept.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wybierz Język</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>197</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jeśli pliki konfiguracyjne nie istnieją, program automatycznie utworzy je w katalogu %Appdata%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>213</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/icons/Resources/Icons/info.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>255</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wybierz Styl</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="First_Theme_Choose">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentText">
    <string notr="true">Fusion</string>
   </property>
   <item>
    <property name="text">
     <string notr="true">Fusion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">Fusion Dark</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">Windows</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wysyłaj anonimowe statystyki</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="First_Statistics">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Nie</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/cancel.png</normaloff>:/icons/Resources/Icons/cancel.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tak</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/icons/Resources/Icons/accept.png</normaloff>:/icons/Resources/Icons/accept.png</iconset>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
